README.md

```diff
@@ -66,9 +66,13 @@ We intend to maintain a linear history on the GitHub `master` branch.
 The project contains a number of tests, used to drive development
 and ensure correctness.  The tests are written using the
 [googletest](https://github.com/google/googletest) framework.  The `googletest`
-source is not provided with this project.  Download the `googletest` source
-into the `<spirv-dir>/external/googletest` directory before configuring
-and building the project.
+source is not provided with this project.  There are two ways to enable
+tests:
+* If SPIR-V Tools is configured as part of an enclosing project, then the
+  enclosing project should configure `googletest` before configuring SPIR-V Tools.
```

Comment:
Well, the enclosing project doesn't _have_ to configure `googletest`; it's just that SPIR-V Tools will find it automatically if it's already configured and won't look for it under `external`.